modal.css

```diff
@@ -0,0 +1,200 @@
+.modal {
+  display: none;
+}
+
+.modal.active {
+  display: block;
+  overflow-y: auto;
+  margin: 21px 6.5% auto 6.5%;
+  inset: 0;
+  border: 1px solid #fff;
+  gap: 24px;
+  position: fixed;
+  z-index: 777;
+  background: rgb(0, 0, 128);
+}
+
+#modal-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 6.5%;
+  margin-right: 6.5%;
+}
+
+#modal-header h1 {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: 38px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.buttons button {
+  padding: 12px;
+  gap: 12px;
+  width: 108px;
+  height: 48px;
+  border-style: none;
+  background: #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
+  display: flex;
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.buttons button:last-child {
+  margin-top: 24px;
+  margin-bottom: 28px;
+  padding: 12px;
+  gap: 12px;
+  width: 124px;
+  height: 48px;
+  background: #7d73fc;
+}
+
+#modal-header button {
+  margin-top: 24px;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  font-size: 1.25rem;
+  color: #fff;
+}
```

Comment:
- [ ] Your `X` buttons have different styling the Figma design kindly make similar to the Figma design
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/175773876-9a3016ad-0f39-4627-aef6-6d63499b52dd.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/175773881-45f48baf-f361-4e7e-8aaa-184b6ed80f3c.png)